Ex4Test.java

```diff
@@ -0,0 +1,40 @@
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class Ex4Test {
+
+    @Test
+    public void testCountIslands(){
+        boolean[][] mapTiles = new boolean[4][4];
+        mapTiles[0][1] = true;
+        mapTiles[1][0] = true;
+        mapTiles[1][1] = true;
+        mapTiles[0][3] = true;
+        mapTiles[2][2] = true;
+        mapTiles[3][2] = true;
+        Ex4 countIslands = new Ex4();

```

Comment:
All the methods are static, so no need to create instance of Ex4. Same in other tests.